app/views/users/new.html.erb

```diff
@@ -1 +1,23 @@
+<% provide :title, t(".title") %>
 <h1><%= t ".title" %></h1>
+
+<div class="row justify-content-md-center">
+  <div class="signup_form col-md-6 col-md-offset-3">
+    <%= render "shared/error_messages" %>
+    <%= form_tag(users_path) do %>
```

Comment:
phân biệt form_for và form_tag

Reply:
- form for tạo form dựa trên model, name của các field phải trùng với tên của attribute trong model
- form tag đơn giản chỉ tạo 1 form cơ bản, khi dùng phải chỉ rõ url và name các input để parse vào param